doc/reference/configuration/cfg_replication.rst

```diff
@@ -313,10 +333,9 @@
 .. confval:: replication_sync_timeout
 
     Since version 1.10.2.
-    The number of seconds that a replica will wait when trying to
-    sync with a master in a cluster,
-    or a :ref:`quorum <cfg_replication-replication_connect_quorum>` of masters,
-    after connecting or during :ref:`configuration update <replication-configuration_update>`.
+    The number of seconds that a node waits when trying to sync with
+    other nodes in a replica set
+    (see :ref:`bootstrap_strategy <cfg_replication-bootstrap_strategy>`).
```

Comment:
I'd also leave the "after connecting or during :ref:`configuration update <replication-configuration_update>`." line here.